tests/testthat/test-unnecessary_nesting_linter.R

```diff
@@ -2,23 +2,29 @@ test_that("unnecessary_nesting_linter skips allowed usages", {
   linter <- unnecessary_nesting_linter()
 
   # parallel stops() and return()s are OK
-  double_stop_lines <- c(
-    "if (A) {",
-    " stop()",
-    "} else {",
-    " stop()",
-    "}"
+  expect_lint(
+    trim_some("
+      if (A) {
+      stop()
```

Comment:
Is the indentation off on purpose?

Reply:
no, good catch... mind fixing it? not sure when I'll get back to it

Reply:
Sure.